### PR TITLE
feat: add clique parameters to genesis

### LINKED ISSUE #169: allow customizing clique parameters
### What feature should we add?

right now unless you set the whole config, you cant customize `{"period": 5, "epoch": 30000}` clique params.
an easy fix is to add them to the fn kwargs with same defaults

### PR DESCRIPTION
**Pull request by antazoey**
### What was wrong?

Closes #169 

### How was it fixed?

add kwargs

### Todo:

- [x] Clean up commit history

- [x] Add or update documentation related to these changes

- [x] Add entry to the [release notes](https://github.com/ethereum/py-geth/blob/main/newsfragments/README.md)

#### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: geth/chain.py [hunk 1]
```diff
@@ -127,7 +129,7 @@ def write_genesis_file(
             # Using the Ethash consensus algorithm is deprecated
             # Instead, use the Clique consensus algorithm
             # https://geth.ethereum.org/docs/interface/private-network
-            "clique": {"period": 5, "epoch": 30000},
+            "clique": {"period": clique_period, "epoch": clique_epoch},
```
Comment (antazoey) [inline geth/chain.py:132]: i just realized shanghai time is also missing

**Comment by antazoey** [comment]
ah i thought i needed this pr but i actually dont, so feel free to disregard

**Comment by kclowes** [comment]
I like these changes, thanks! I also added `shanghaiBlock`.

**Review by kclowes — APPROVED** [review]
Thanks @antazoey!

**Pull request merged**